gcp_variant_transforms/libs/bigquery_util.py

```diff
@@ -357,12 +358,18 @@ def calculate_optimal_range_interval(range_end):
     * _RANGE_INTERVAL_SIG_DIGITS is set to 1 which adds [0, 10^1 * 3999)
   In total we add [10^4, 10 * 3999 + 2 * 10^4) buffer to range_end.
 
+  range_end must be capped at MAX_RANGE_END = pow(2, 63) - 1 which is required
+  by BigQuery integer range partitioning.
+
   Args:
     range_end: the maximum value of the column subject to partitioning
 
   Returns:
     A tuple (partition size, partition size * 3999).
   """
+  if range_end >= MAX_RANGE_END:
+    return(int(MAX_RANGE_END / float(_MAX_BQ_NUM_PARTITIONS)),
```

Comment:
Hmm, I see what you are doing here, but can't we find a suitable name for the constant MAX_RANGE_END/_MAX_BQ_NUM_PARTITIONS? it's not really max_interval, but if we can find a suitable name, we should move it.